packages/app/src/vis-packs/core/line/MappedLineVis.tsx

```diff
@@ -26,7 +25,9 @@ interface Props {
   value: NumArray;
   valueLabel?: string;
   errors?: NumArray;
-  auxiliaries?: Auxiliary[];
+  auxLabels?: string[];
+  auxValues?: NumArray[];
+  auxErrors?: (NumArray | undefined)[];
```

Comment:
Passing everything in separate arrays for consistency with axes values/labels. I'm still not hugely satisfied with the mapped components' APIs but by clearly separating values from metadata in the containers, we're a step closer to seeing a good refactoring, I reckon.